code/modules/mob/living/carbon/alien/adult/alien_powers.dm

```diff
@@ -338,32 +338,31 @@ Doesn't work on other aliens/AI.*/
 	new choice_path(owner.loc)
 	return TRUE
 
-/datum/action/cooldown/alien/sneak
+/datum/action/cooldown/sneak/alien
```

Comment:
Make sure this stuff caries over from the old imlpementation
`panel = "Alien"`
`check_flags = AB_CHECK_IMMOBILE | AB_CHECK_CONSCIOUS | AB_CHECK_INCAPACITATED`